events/wol_intrigue_events.txt

```diff
@@ -668,7 +668,7 @@ character_event = {
 				character_event = { id = WoL.2080 }
 			}
 			
-			3000 = { # Chance to slander
```

Comment:
This is not province ID

Reply:
Fixed in 6d39aee5f58d4d8fbe7faf298e3a9d6e83b76d60.